src/query.rs

```diff
@@ -252,28 +281,32 @@ mod tests {
                         )
                     };
 
-                assert!(
-                    run_query_on(&indexed_crate_new, &indexed_crate_new)
-                        .1
-                        .is_empty(),
-                    "{}",
-                    crate_pair
-                );
-                assert!(
-                    run_query_on(&indexed_crate_old, &indexed_crate_old)
-                        .1
-                        .is_empty(),
-                    "{}",
-                    crate_pair
-                );
+                let assert_no_false_positives_in_nonchanged_crate =
+                    |crate_: &VersionedIndexedCrate,
+                     crate_version: &str| {
+                         let output_pair = run_query_on(crate_, crate_);
+                         if !output_pair.1.is_empty() {
```

Comment:
nit: `output_pair.1` is opaque. You already have a `let` binding, and destructuring the tuple gives you the opportunity to give `output_pair.1` a name that will guide the reader's intuition.

Code is written ~once but read many, many times, by many different people, most of whom didn't write or review it originally. Optimize for readability, not for being easy to write.

Reply:
Fixed in a22b9b5a1123262abdc190c3dc561ce2365e8084